util/chplenv/chpl_comm.py

```diff
@@ -24,7 +24,7 @@ def get():
                 compiler_val in ugni_comp):
             comm_val = 'ugni'
         # automatically uses gasnet when on a cray-x* or cray-cs machine
-        elif platform_val.startswith('cray-'):
+        elif platform_val.startswith('cray-') or platform_val.startswith('cross-'):
```

Comment:
I think aarch64 should default to CHPL_COMM=none until the later date when multilocale is supported.